src/central_control/scripts/central_control.py

```diff
@@ -432,13 +440,35 @@ def handle_object_recognition(self, res: DetectionResults):
 
         detection_list: List[DetectionResult] = res.detection_results
         for detection in detection_list:
-            if detection.class_num == 11 and detection.depth < 40 and detection.confidence > 0.4:
+            '''if detection.class_num == 11 and detection.depth < 40 and detection.confidence > 0.4:
                 # Stop sign. Depth is likely going to be smaller than 25 because of the image resolution
-                self.sign_data.append(detection)
-            elif detection.class_num == 2 and detection.confidence > 0.4:
+                self.sign_data.append(detection)'''
+            if detection.class_num == 2 and detection.confidence > 0.4:
                 # Only deal with cars for now
                 self.object_data.append(detection)
 
+    def handle_stop_sign(self, res: DetectionResults):
+        if not self.ready:
+            self.ready = True
+        # Sort through and deal with sign detection
+        self.sign_data.clear()
+        detection_list: List[DetectionResult] = res.detection_results
+        for detection in detection_list:
+            if detection.class_num == 11 and detection.depth < 40 and detection.confidence > 0.4:
+                # Stop sign. Depth is likely going to be smaller than 25 because of the image resolution
+                self.sign_data.append(detection)
+
+    '''def handle_stop_sign(self, res: DetectionResults):
+        if not self.ready:
+            self.ready = True
+        # Sort through and deal with sign detection
+        self.sign_data.clear()
+        detection_list: List[DetectionResult] = res.detection_results
+        for detection in detection_list:
+            if detection.class_num == 11 and detection.depth < 50 and detection.confidence > 0.2:
+                # Stop sign. Depth is likely going to be smaller than 25 because of the image resolution
+                self.sign_data.append(detection)'''
```

Comment:
you can delete this as the method above is pretty much identical

Reply:
Fair enough, gonna do that